test/functional/wallet_migration.py

```diff
@@ -470,6 +471,40 @@ def test_unloaded_by_path(self):
 
         assert_equal(bals, wallet.getbalances())
 
+    def test_default_wallet(self):
+        self.log.info("Test migration of the default wallet")
```

Comment:
In commit "test: Add test for migrating default wallet and plain file wallet" (52634dba1cba8928b8767d05c8e30b5fd3da45e4)

Maybe say "legacy top level wallet" instead of "default wallet". Default wallet isn't really a concept that exists after  #15454

Reply:
Will do if I need to push again.